### ISSUE TITLE
does it work on OSX Sierra? 

### ISSUE THREAD

**Issue by bkarlson** [FabriceSalvaire/CodeReview#21]
Trying to run pyqgit, getting the following:

``` bash
$ pyqgit
2017-11-01 20:43:10,275 - pyqgit - Logging.setup_logging - INFO - Start pyqgit
2017-11-01 20:43:10,304 - CodeReview.GUI.LogBrowser.LogBrowserApplication - ApplicationBase.__init__ - DEBUG - Namespace(path='/Users/work', user_script=None, user_script_args='') {}
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.6/bin/pyqgit", line 67, in <module>
    application = LogBrowserApplication(args)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/CodeReview/GUI/LogBrowser/LogBrowserApplication.py", line 46, in __init__
    super(LogBrowserApplication, self).__init__(args=args)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/CodeReview/GUI/Base/GuiApplicationBase.py", line 58, in __init__
    super(GuiApplicationBase, self).__init__(args=args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/CodeReview/Application/ApplicationBase.py", line 49, in __init__
    self._platform = Platform()
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/CodeReview/Tools/Platform.py", line 48, in __init__
    self.os = self._get_os()
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/CodeReview/Tools/Platform.py", line 85, in _get_os
    raise RuntimeError('unknown platform')
RuntimeError: unknown platform
QWidget: Must construct a QApplication before a QWidget
```
Installed packages:

``` bash
$ pip3 list
cffi (1.11.2)
CodeReview (0.3.1)
pip (9.0.1)
pycparser (2.18)
pygit2 (0.26.0)
Pygments (2.2.0)
PyQt5 (5.9)
PyYAML (3.12)
setuptools (28.8.0)
sip (4.19.3)
six (1.11.0)
```

**Comment by FabriceSalvaire** [comment]
Yes, it must work, but code must be fixed.

**Comment by FabriceSalvaire** [comment]
fixed 354911dc939bae07706c7965eae4f2450b743595
